src/server/game/Entities/Player/Player.cpp

```diff
@@ -4623,18 +4623,25 @@ void Player::ResurrectPlayer(float restore_percent, bool applySickness)
     //for each level they are above 10.
     //Characters level 20 and up suffer from ten minutes of sickness.
     int32 startLevel = sWorld->getIntConfig(CONFIG_DEATH_SICKNESS_LEVEL);
+    const ChrRacesEntry* raceEntry = sChrRacesStore.LookupEntry(GetRace());
```

Comment:
`ChrRacesEntry const* raceEntry = sChrRacesStore.AssertEntry(GetRace());`

* const placement (codestyle issue)
* Use assert and remove the !raceEntry check, it can never fail when player is in world and ResurrectPlayer happens

Reply:
Okie dokie, it should be corrected now.